### PR TITLE
Fix rcmd checks

### PR DESCRIPTION
**Pull request by Robert-Krajcik**
Addresses Issue #263 Clear out R CMD Check issues

### CODE REVIEW

File: .Rbuildignore [hunk 1]
```diff
@@ -8,3 +8,4 @@ $credentials.*
 $database.*
 ^README\.Rmd$
 ^\.github$
+^LICENSE\.md$
```
Comment (Jeff-Thompson12) [inline .Rbuildignore:11]: It feels like we should add some items to `.Rbuildignore` to knock out a couple notes.

image

**Review by Jeff-Thompson12 — COMMENTED** [review]
Is there any way to determine what functions (if any) are being used by `{crayon}`, `{desc}`, & `{forcats}`?

image

File: .Rbuildignore (hunk 1 above)
Comment (Robert-Krajcik) [reply .Rbuildignore:11]: Yeah I think I can find which functions from `crayon`, `desc` and `forcats` are being used here.
We're getting rid of `www`
And I can add some files to .gitignore

**Review by AARON-CLARK — DISMISSED** [review]
Hi @Robert-Krajcik, looks great! Thanks for the PR. The only note I have is that all the changes you made the the `README.md` file will likely be overwritten the next time someone renders the `README.rmd` file. We may even start using a GitHub Action workflow to automatically render it for us. If you think you can improve the Rmd file, feel free to amend it in a separate PR. I'm going to merge this one!

**Pull request merged**